spec/dummy/config/webpack/serverWebpackConfig.js

```diff
@@ -109,6 +111,31 @@ const configureServer = () => {
   // If using the React on Rails Pro node server renderer, uncomment the next line
   // serverWebpackConfig.target = 'node'
 
+  // MiniRacer specific config
+  if (process.env.EXECJS_RUNTIME === 'MiniRacer') {
```

Comment:
```suggestion
  if (process.env.MANUAL_EXECJS_RUNTIME === 'MiniRacer') {
```

Reply:
I have no idea what this change is about.

See:
https://github.com/search?q=repo%3Arails%2Fexecjs%20EXECJS_RUNTIME&type=code

Reply:
I explained this webpack code in the github comment above:
> Miniracer needs a couple polyfills for some Node.js modules that are used in React 18 SSR

Also, EXECJS_RUNTIME doesn't work (we're using some different name, MANUAL_EXECJS_RUNTIME) because it breaks when trying to autoload the Alaska gem